src/Raven.Server/Config/Categories/ShardingConfiguration.cs

```diff
@@ -32,8 +32,8 @@ public ShardingConfiguration()
         [Description("Enable the timeout of the orchestrator's requests to the shards")]
         [DefaultValue(DefaultValueSetInConstructor)]
         [TimeUnit(TimeUnit.Minutes)]
-        [ConfigurationEntry("Sharding.OrchestratorTimeoutInMinutes", ConfigurationEntryScope.ServerWideOnly)]
-        public TimeSetting OrchestratorTimeoutInMinutes { get; set; }
+        [ConfigurationEntry("Sharding.OrchestratorTimeoutInMin", ConfigurationEntryScope.ServerWideOnly)]
+        public TimeSetting OrchestratorTimeoutInMin { get; set; }
```

Comment:
`OrchestratorTimeoutInMin` -> `OrchestratorTimeout`

The configuration key (the string) needs to have a unit
But since property is a timesetting, it does not :)

Reply:
Fixed :)